rollbar/__init__.py

```diff
@@ -445,6 +472,12 @@ def search_items(title, return_fields=None, access_token=None, endpoint=None, **
                     **search_fields)
 
 
+def wait(f=None):
+    _threads.join()
```

Comment:
This join will return when all `task_done` is called on the queue the same number of times that put has been called